webapp/ext/controller/ListReportExt.controller.js

```diff
@@ -0,0 +1,69 @@
+sap.ui.define([
+    "sap/ui/model/Filter", 
+    "sap/ui/comp/smartfilterbar/SmartFilterBar", 
+    "sap/m/DateRangeSelection",
+	"../mixin/functionImports"
+], function (Filter,
+	SmartFilterBar,
+	DateRangeSelection,
+    functionImports) {
+    "use strict";
+    return {
```

Comment:
spread functionImports so you don't need to invoke .call(this)

...functionImports